submissions/ses3332002/dom/script.js

```diff
@@ -0,0 +1,257 @@
+      "Burn This Disco Out"
+    ],
+    "bgColor": 19
+  },
+  {
+    "title": "Thriller (1982)",
+    "cover": "thriller",
+    "songs": [
+      "Wanna Be Startin' Somethin'",
+      "Baby Be Mine",
+      "The Girl Is Mine",
+      "Thriller",
+      "Beat It",
+      "Billie Jean",
+      "Human Nature",
+      "P.Y.T. (Pretty Young Thing)",
+      "The Lady in My Life"
+    ],
+    "bgColor": 18
+  },
+  {
+    "title": "Bad (1987)",
+    "cover": "bad",
+    "songs": [
+      "Bad",
+      "The Way You Make Me Feel",
+      "Speed Demon",
+      "Liberian Girl",
+      "Just Good Friends",
+      "Another Part of Me",
+      "Man in the Mirror",
+      "I Just Can't Stop Loving You",
+      "Dirty Diana",
+      "Smooth Criminal",
+      "Leave Me Alone"
+    ],
+    "bgColor": 0
+  },
+  {
+    "title": "Dangerous (1991)",
+    "cover": "dangerous",
+    "songs": [
+      "Jam",
+      "Why You Wanna Trip on Me",
+      "In the Closet",
+      "She Drives Me Wild",
+      "Remember the Time",
+      "Can't Let Her Get Away",
+      "Heal the World",
+      "Black or White",
+      "Who Is It",
+      "Give in to Me",
+      "Will You Be There",
+      "Keep the Faith",
+      "Gone Too Soon",
+      "Dangerous"
+    ],
+    "bgColor": 29
+  },
+  {
+    "title": "HIStory: Past, Present and Future, Book I (1995)",
+    "cover": "history",
+    "songs": [
+      "Scream",
+      "They Don’t Care About Us",
+      "Stranger in Moscow",
+      "This Time Around",
+      "Earth Song",
+      "D.S.",
+      "Money",
+      "Come Together",
+      "You Are Not Alone",
+      "Childhood",
+      "Tabloid Junkie",
+      "2 Bad",
+      "HIStory",
+      "Little Susie",
+      "Smile"
+    ],
+    "bgColor": 332
+  },
+  {
+    "title": "Invincible (2001)",
+    "cover": "invincible",
+    "songs": [
+      "Unbreakable",
+      "Heartbreaker",
+      "Invincible",
+      "Break of Dawn",
+      "Heaven Can Wait",
+      "You Rock My World",
+      "Butterflies",
+      "Speechless",
+      "2000 Watts",
+      "You Are My Life",
+      "Privacy",
+      "Don't Walk Away",
+      "Cry",
+      "The Lost Children",
+      "Whatever Happens",
+      "Threatened"
+    ],
+    "bgColor": 215
+  },
+  {
+    "title": "Michael (2010)",
+    "cover": "michael",
+    "songs": [
+      "Hold My Hand",
+      "Hollywood Tonight",
+      "Keep Your Head Up",
+      "(I Like) The Way You Love Me",
+      "Monster",
+      "Best of Joy",
+      "Breaking News",
+      "(I Can't Make It) Another Day",
+      "Behind the Mask",
+      "Much Too Soon"
+    ],
+    "bgColor": 8
+  },
+  {
+    "title": "Xscape (2014)",
+    "cover": "xscape",
+    "songs": [
+      "Love Never Felt So Good",
+      "Chicago",
+      "Loving You",
+      "A Place with No Name",
+      "Slave to the Rhythm",
+      "Do You Know Where Your Children Are",
+      "Blue Gangsta",
+      "Xscape"
+    ],
+    "bgColor": 33
+  }
+];
+
+let doc = document.body;
```

Comment:
Why did you use `let` here?

Reply:
Do you mind "why let but not const"?
Nothing criminal, just a habit.
Well, I can remake this.

Reply:
I had the same habit)) but it's better to use their exact purpose

Reply:
Just a good practice. Usually you will have linter rule which will tell you to use `const` if you will not reassign variable.

Reply:
I'ii remake it.

Reply:
ready